src/webview-contract/webviewDefinitions/connectAcrToCluster.ts

```diff
@@ -0,0 +1,93 @@
+import { WebviewDefinition } from "../webviewTypes";
+
+export type SubscriptionKey = {
+    subscriptionId: string;
+};
+
+export type Subscription = SubscriptionKey & {
+    name: string;
+};
+
+export type AcrKey = SubscriptionKey & {
+    resourceGroup: string;
+    acrName: string;
+};
+
+export type Acr = AcrKey; // Fully-defined by its key
+
+export type ClusterKey = SubscriptionKey & {
+    resourceGroup: string;
+    clusterName: string;
+};
+
+export type Cluster = ClusterKey; // Fully-defined by its key
+
+// https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles
+export const acrPullRoleDefinitionName = "7f951dda-4ed3-4680-a7ca-43fe172d538d";
```

Comment:
I'm adding `// Role definition names are always guids. Here we want the role definition name for AcrPull.`

Adding this because even though I feel like the variable name is already pretty explanatory, I don't feel _very_ strongly either way.